xapian-bindings/configure.ac

```diff
@@ -116,15 +116,15 @@ INTREE_DYLD_PATH=
 case $host_os in
 darwin*)
   NEED_JNILIB_EXT=yes
-  dnl If we're building against an uninstalled xapian-core, xapian-config
+  dnl If we're building against an uninstalled xapian-letor, xapianletor-config
   dnl --libs will give an error.
-  if $XAPIAN_CONFIG --libs >/dev/null 2>/dev/null ; then
+  if $XAPIANLETOR_CONFIG --libs >/dev/null 2>/dev/null ; then
```

Comment:
Surely we need to do both? Because we want to be able to work against an uninstalled core _and_ an uninstalled letor.